applications/StructuralMechanicsApplication/custom_elements/solid_elements/base_solid_element.cpp

```diff
@@ -748,6 +748,10 @@ void BaseSolidElement::CalculateOnIntegrationPoints(
     if ( rOutput.size() != number_of_integration_points )
         rOutput.resize( number_of_integration_points, false );
 
+    for (IndexType i_gauss = 0; i_gauss < number_of_integration_points; ++i_gauss) {
+        rOutput[i_gauss] = 0.0;
+    }
+
```

Comment:
I think this is a hotfix that affects any solid element. I'd make a fast PR only for this (just to merge it as fast as possible and to ease any eventual backtraking).

Reply:
ok! incoming

Reply:
https://github.com/KratosMultiphysics/Kratos/pull/13015

Reply:
Shouldn't these changes belong to that PR?